apps/nowait-admin/src/components/closeButton.tsx

```
@@ -0,0 +1,15 @@
+import React from "react";
+import closeIcon from "../assets/close.svg";
+interface CloseButtonProps {
+  onClick?: () => void | undefined;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)